db/schema.rb

```diff
@@ -11,25 +11,32 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20151018203903) do
+ActiveRecord::Schema.define(version: 20151020033930) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "groups", force: :cascade do |t|
     t.string   "name"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at",      null: false
+    t.datetime "updated_at",      null: false
+    t.integer  "neighborhood_id"
   end
 
+  add_index "groups", ["neighborhood_id"], name: "index_groups_on_neighborhood_id", using: :btree
+
   create_table "neighborhoods", force: :cascade do |t|
     t.string   "title"
     t.text     "text"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.string   "location"
+    t.string   "user"
```

Comment:
@HacksForHugs This shouldn't be here. This should be in the neighborhood to user join table.